### PR TITLE
goenv 1.0.0

### PR DESCRIPTION
**Pull request by syndbg**
Changes (last is most recent):

* syndbg/goenv@e11de94
* syndbg/goenv@ce93f67

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Folow-up of  #11220

### CODE REVIEW

**Comment by syndbg** [comment]
`audit` does not pass since I change to SemVer

File: Formula/goenv.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Goenv < Formula
   desc "Go version management"
   homepage "https://github.com/syndbg/goenv"
-  url "https://github.com/syndbg/goenv/archive/v20161215.tar.gz"
-  sha256 "b2fb6b0dc2a359caa628e7ceee3e5c46205d41bb185d461a1c591bd119f84286"
+  url "https://github.com/syndbg/goenv/archive/1.0.0.tar.gz"
+  sha256 "5eac938fc76951e6d3a618bc2ba0832524c5b583dcaeda73be74b9315bf18e0c"
```
Comment (fxcoudert) [inline Formula/goenv.rb]: Add a line with `version_scheme 1` after the `sha256` line

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/goenv.rb (hunk 1 above)
Comment (syndbg) [inline Formula/goenv.rb]: Added, but `audit` is still not happy.

**Review by fxcoudert — APPROVED** [review]

File: Formula/goenv.rb (hunk 1 above)
Comment (fxcoudert) [inline Formula/goenv.rb]: Actually an `audit` bug…

**Comment by fxcoudert** [comment]
Thanks @syndbg for your contribution to Homebrew!

**Comment by syndbg** [comment]
@fxcoudert My pleasure. Thanks for the help!

**Pull request merged**